scripts/lib/api/updateLinks.ts

```diff
@@ -95,10 +99,29 @@ export function updateUrl(
   return url;
 }
 
+export function relativizeLink(link: Link): Link | undefined {
+  const prefixes = [
+    "https://qiskit.org/documentation/apidoc/",
+    "https://qiskit.org/documentation/stubs/",
+  ];
+  const prefix = prefixes.find((prefix) => link.url.startsWith(prefix));
+  if (!prefix) {
+    return;
+  }
+  let [url, anchor] = link.url.split("#");
+  url = removePrefix(url, prefix);
+  url = removeSuffix(url, ".html");
+  if (anchor && anchor !== url) {
+    url = `${url}#${anchor}`;
+  }
+
+  const newText = link.url === link.text ? url : undefined;
```

Comment:
We used to have a boolean variable called `updateText` for the condition. I inlined it.